pyscriptjs/tests/integration/test_py_config.py

```diff
@@ -0,0 +1,43 @@
+import tarfile
+from io import BytesIO
+
+import requests
+
+from .support import PyScriptTest
+
+URL = "https://github.com/pyodide/pyodide/releases/download/0.20.0/pyodide-build-0.20.0.tar.bz2"
+
+
+def download_and_unzip(url, extract_to="."):
+    response = requests.get(url, stream=True)
+    file = tarfile.open(fileobj=BytesIO(response.raw.read()), mode="r:bz2")
+    file.extractall(path=extract_to)
+
+
+class TestPyConfig(PyScriptTest):
+    def test_config(self):
+        download_and_unzip(
+            url=URL,
+            extract_to=self.tmpdir,
+        )
+
+        self.pyscript_run(
+            snippet="""
+            <py-script>
+                import sys
+                sys.modules["pyodide"].__version__
+            </py-script>
+        """,
+            config="""
+            <py-config>
+                autoclose_loader: true
+                runtimes:
+                - src: "/pyodide/pyodide.js"
+                  name: pyodide-0.20.0
+                  lang: python
+            </py-config>
+        """,
+        )
+
+        version = self.page.locator("py-script").inner_text()
+        assert version == "0.20.0"
```

Comment:
This is a good test, I like it!
A few notes to improve it:
1. the name of the test is wrong: what it actually tests is that the `runtimes:` config works as intended, so it should be called `test_runtimes`
2. you should add a comment which says that the default pyodide version is newer and that here we check that we are able to load a different version
3. This is a nitpick, but I think it would be better to write to `console.log` instead than to the HTML in the page. The small advantage is that if you print to the console, you can see the output also in the pytest logs in the terminal and most importantly in the CI logs, so it will make things easier to debug in case it fails in the future.
4. `autoclose_loader` is not needed

Reply:
1. I renamed it to `test_runtime_config` since that is more appropriate.
2. done, added it as a docstring though
3. done, I have written to `console.info`, but also preserved writing to HTML in the page. I chose `console.info` since it is easier to test that using `self.console.info.lines` because `self.console.log.lines` contains a lot of stuff already
4. removed `autoclose_loader`

Reply:
+1 to use `console.info` instead of `console.log`, we already do it for other tests.

I don't really understand why you *also* need to write to the HTML, but I don't care enough to block the PR for this :)